starters/next12-react-query-tailwind/README.md

```diff
@@ -1,4 +1,4 @@
-# next-react-query-tailwind starter kit
+# next12-react-query-tailwind starter kit
 
 This starter kit features **Next.js**, **React Query**, and **Tailwind CSS**.
```

Comment:
I think this line should be change too so it says Next.js v.12 or something like that.